src/pages/index.js

```diff
@@ -87,68 +94,91 @@ const HomePage = ({ data }) => {
+              link={link}
+            />
           ))}
         </DocTileGrid>
       </Section>
       <Section layout={layout}>
-        <SectionTitle title={fso.title} icon="nr-fso" />
-        <SectionDescription>{fso.description}</SectionDescription>
+        <SectionTitle title={t('home.fso.title')} icon="nr-fso" />
+        <SectionDescription>{t('home.fso.description')}</SectionDescription>
         <DocTileGrid>
-          {fso.tiles.map((guide, idx) => (
-            <DocTile key={idx} guide={guide} />
+          {fso.tiles.map((link, idx) => (
+            <DocTile
+              key={idx}
+              title={t(`home.fso.t${idx + 1}.title`)}
+              description={t(`home.fso.t${idx + 1}.description`)}
+              link={link}
+            />
           ))}
         </DocTileGrid>
       </Section>
       <Section alternate layout={layout}>
-        <SectionTitle title={ai.title} icon="nr-ai" />
-        <SectionDescription>{ai.description}</SectionDescription>
+        <SectionTitle title={t('home.ai.title')} icon="nr-ai" />
+        <SectionDescription>{t('home.ai.description')}</SectionDescription>
         <DocTileGrid>
-          {ai.tiles.map((guide, idx) => (
-            <DocTile key={idx} guide={guide} />
+          {ai.tiles.map((link, idx) => (
+            <DocTile
+              key={idx}
+              title={t(`home.ai.t${idx + 1}.title`)}
+              description={t(`home.ai.t${idx + 1}.description`)}
+              link={link}
+            />
           ))}
         </DocTileGrid>
       </Section>
       <Section layout={layout}>
         <SectionTitle title="New Relic integrations" />
         <SectionDescription>
-          <Link to="/docs/integrations/intro-integrations/get-started/introduction-new-relic-integrations">
-            Integrations
-          </Link>{' '}
-          connect the technologies in your stack to New Relic. Here are a few of
-          our{' '}
-          <a href="https://newrelic.com/integrations" rel="noopener noreferrer">
-            370+ integrations
-          </a>
-          :
+          <Trans i18nKey="home.integrations.description">
+            <Link to="/docs/integrations/intro-integrations/get-started/introduction-new-relic-integrations">
+              Integrations
+            </Link>{' '}
+            connect the technologies in your stack to New Relic. Here are a few
+            of our{' '}
+            <a
+              href="https://newrelic.com/integrations"
+              rel="noopener noreferrer"
+            >
+              370+ integrations
+            </a>
+            :
+          </Trans>
         </SectionDescription>
 
-        {integrations.map(({ title, tiles }, idx) => (
+        {integrations.map((integration, idx) => (
           <Fragment key={idx}>
-            <IntegrationTitle>{title}</IntegrationTitle>
+            <IntegrationTitle>
+              {t(`home.integrations.h${idx + 1}`)}
```

Comment:
[nit] Could we name this key `home.integrations.title${idx}`? The `h` makes me think its named after an `h1`/`h2` tag, which was a bit misleading in my brain 😅

Reply:
Yeah, I thought that could be wonky. I chose `h` because of heading, but that's certainly confusing. I'll update it with your suggestion.